core/src/main/java/org/jboss/pnc/causeway/brewclient/BrewClientImpl.java

```diff
@@ -232,8 +232,12 @@ private boolean checkImportErrors(KojiImportResult result, ImportFileGenerator i
         if (kojiErrors != null) {
             for (Map.Entry<String, KojijiErrorInfo> e : kojiErrors.entrySet()) {
                 String artifactId = importFiles.getId(e.getKey());
-                log.warn("Failed to import artifact {} ({}): {}", artifactId, e.getKey(), e.getValue());
-                log.debug("Artifact import error:", e);
+                if (log.isWarnEnabled()) {
+                    KojijiErrorInfo errorInfo = e.getValue();
+                    String message = String
+                            .format("Failed to import artifact %s (%s): %s", artifactId, e.getKey(), errorInfo);
+                    log.warn(message, errorInfo.getError());
```

Comment:
Is KojijiErrorInfo an exception?

Reply:
No, it's a DTO. But KojijiErrorInfo.error is.